src/utils/openDownloadApp.ts

```diff
@@ -0,0 +1,42 @@
+import { sendAnalyticsEvent } from '@uniswap/analytics'
+import { InterfaceElementName, InterfaceEventName, SharedEventName } from '@uniswap/analytics-events'
+import { isIOS } from 'utils/userAgent'
+
+const APP_STORE_LINK = 'https://apps.apple.com/app/apple-store/id6443944476'
+const MICROSITE_LINK = 'https://wallet.uniswap.org/'
+
+type OpenDownloadAppOptions = {
+  appStoreParams?: string
+  microSiteParams?: string
+  analyticsEventName?: SharedEventName
+}
+
+const defaultDownloadAppOptions = {
+  appStoreParams: `?pt=123625782&ct=In-App-Banners&mt=8`,
+  analyticsEventName: SharedEventName.ELEMENT_CLICKED,
+}
+
+export function openDownloadApp(
+  element?: InterfaceElementName,
+  options: OpenDownloadAppOptions = defaultDownloadAppOptions
+) {
+  sendAnalyticsEvent(options.analyticsEventName ?? SharedEventName.ELEMENT_CLICKED, { element })
```

Comment:
Will this just log an element clicked event with no properties if there's no element? Should we only log when there's an element?

Reply:
good question, i defer to @tinaszheng

Reply:
I think it will in this case - once the analytics PR merges we should make sure this doesn't allow an undefined analytics event here / doesn't log any event in that case.